common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -73,9 +73,13 @@ public class StringUtils {
 
     public static final Gson gson;
 
+    public static final Gson plainDoubleGson;
```

Comment:
Thanks for making the changes at StringUtils and adding tests accordingly, LGTM!